common/src/main/public/development.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "environment": "DEVELOPMENT",
-  "odbURI": "wss://lucuma-odb-development.herokuapp.com/ws",
+  "odbURI": "wss://lucuma-odb-master.herokuapp.com/ws",
```

Comment:
Should this be `staging`?

Reply:
It should actually be `development`, but that's still in the old API. After we merge this, we can publish the latest API to `lucuma-odb-development` and remove `lucuma-odb-master`.